deploy.spryker-b2b-staging.yml

```diff
@@ -0,0 +1,221 @@
+version: "0.1"
```

Comment:
depending on TW needs we will remove this one after smoke testing, or keep it as it is to refer in the documentation